tasks/push-to-cdn/tests/test-push-to-cdn.yaml

```diff
@@ -0,0 +1,84 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-push-to-cdn
+spec:
+  description: |
+    Run the push-to-cdn task and verify the results
+  workspaces:
+    - name: tests-workspace
+  tasks:
+    - name: setup
+      workspaces:
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        workspaces:
+          - name: data
+        steps:
+          - name: setup-values
+            image: quay.io/redhat-appstudio/release-service-utils:bc81bfed6062a386e48a76b252c6f33b52c411b0
```

Comment:
Can you adjust this to a konflux-ci/release-service-utils image? Any other task should have one to copy from (including your check-result below)